src/sdk/pynni/nni/nas/pytorch/fixed.py

```diff
@@ -0,0 +1,78 @@
+import json
```

Comment:
why this filename `fixed.py` is so weird?

Reply:
Any suggestions?